source/Halibut/Transport/SecureListener.cs

```diff
@@ -40,22 +40,22 @@ public class SecureListener : IDisposable
         readonly Func<Dictionary<string, string>> getFriendlyHtmlPageHeaders;
         readonly CancellationTokenSource cts = new CancellationTokenSource();
         readonly TcpClientManager tcpClientManager = new TcpClientManager();
-        readonly ExchangeActionAsync exchangeAction;
```

Comment:
Is `ExchangeActionAsync` unused now or is it still used in other places?

If it's unused then `ExchangeActionWithCancellationAsync` -> `ExchangeActionAsync`

Reply:
Yep, it's unused now. I will remove and rename as appropriate. 👍🏻